bin/crew

```diff
@@ -1328,6 +1328,14 @@ def install
     puts "Package #{@pkg.name} already installed, skipping...".lightgreen
     return
   end
+    
+  if @pkg.deprecated?
+    puts "Package #{@pkg.name} is deprecated".orange
+    if @pkg.in_upgrade
+       puts "\b, skipping...".orange
+       return
+    end
+  end
```

Comment:
Just take this out... We would want to add a boolean for this in `lib/package.rb` in any case before we did this... but I don't think marking this with a deprecated flag when some larger proportion of our software might be abandoned upstream...